src/raftstore/store/peer.rs

```diff
@@ -1210,6 +1365,32 @@ impl Peer {
         true
     }
 
+    fn maybe_propose_normal(&mut self,
+                            meta: ProposalMeta,
+                            cb: Callback,
+                            mut req: RaftCmdRequest,
```

Comment:
If the request is `ComputeHash`, must flush call `flush_pending_proposal`, at the same time, `ComputeHash` can not share the same proposal batch with other commands.